src/lib/types/restrictions.d.ts

```diff
@@ -9,6 +9,7 @@ declare namespace Restrictions {
   export type Restrictions = Restriction[]
   interface Restriction {
     None?: None
+    op?: "AND" | "OR" | null
```

Comment:
missing ops?

Reply:
indeed

Reply:
added, and will be pushed with the latest commit. thanks for flagging up

Reply:
Pushed.